scripts/install-mender.sh

```diff
@@ -625,6 +628,9 @@ check_dist_and_version() {
                 fi
                 DIST_VERSION="$(sed 's/\/.*//' /etc/debian_version | sed 's/\..*//')"
                 case "$DIST_VERSION" in
+                    13)
+                        DIST_VERSION="trixie"
+                    ;;
                     12)
                         DIST_VERSION="bookworm"
                     ;;
```

Comment:
Looks good!

FYI @elkoniu When doing a `master` -> `production` update the only thing that really gets updated is this script.